articles/choosing-between-firebase-and-mongodb/index.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: engineering-education
+status: publish
+published: true
+slug: choosing-between-firebase-and-mongo-db
+title: Choosing Between Firebase and MongoDB
+description: Respective history and pros and cons of using Firebase or MongoDB in a tech stack.
+date: 2020-11-09T00:00:00-07:00
+topics: [databases][mongodb][firebase]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/choosing-between-firebase-and-mongo-db/hero.jpg
+    alt: Relational Databases
+---
+
+### MongoDB vs Firebase
+Databases play an important role in determining the functionality and quality of an application or a website. For an app to be successful,
+the developer will have to consider the users of the application or the website.
```

Comment:
Spacing irregularity

Reply:
"The developer will have to consider the users of the application of the website" => very generic. Replace instead with why databases are crucial to app development to give some context.